### ISSUE TITLE
pcg website's pcg-c download link: use github's automated zipball

### ISSUE THREAD

**Issue by listx** [imneme/pcg-c#6]
We should probably use git to tag a release, and then provide 2 links on the website: (1)  zipball for the tagged release (something like https://help.github.com/articles/linking-to-releases/) and (2) zipball for master.

(2) is easy to do right away because it already exists: https://github.com/imneme/pcg-c/zipball/master.

(1) requires creating a tagged commit, then pushing it here, and then linking it using the 'zipball' syntax somehow.

I just think it's better to do this as it is more automated and also offloads the storage/bandwidth (however trivial) from pcg-random.org to github.

**Comment by imneme** [comment]
This is indeed true and on the to-be-fixed list.

**Comment by Ivoz** [comment]
@imneme it seems you've done some great work incorporating changes recently [thanks!]; maybe you would like to consider applying this issue as a finishing step now?

Apply a tag(s) (0.99? 1.0? whatever, really) for the current code state, and/or also do a github release (where you can upload arbitary codeballs if you like) and then I would think you could update your [website](http://www.pcg-random.org/download.html) to point to the new code?

https://help.github.com/en/articles/creating-releases would be useful if you would like to keep the previous convention on your website where you released different zips for different language compatibility. Otherwise a tag / release of the current codebase would be fine.

**Comment by imneme** [comment]
Yes, it's something I'll be moving to do in the next week or so.  Feel free to prod me if it's not done in a timely way.

**Comment by Ivoz** [comment]
@imneme you said you wouldn't mind a prod, so here's one :D

**Comment by scottchiefbaker** [comment]
Poke... is this still an option? I would appreciate this.
